CONTRIBUTING.md

```diff
@@ -1,4 +1,41 @@
-## Releasing a new version
+# Development
+
+Set up an alias for the EAS CLI so you can try it in projects all around your computer. The project is compiled on the fly so you don't need to run a build (watch) command. Open your `.zshrc` or other config file and add:
```

Comment:
only eas-cli is compiled automatically, you still need to run `yarn build` at least once to build eas-json